src/views/Home/components/GuestWelcome/index.js

```diff
@@ -1,32 +1,110 @@
-import { Card, CardContent, CardHeader, Grid, Typography } from '@material-ui/core';
-import React from 'react';
-import Login from 'views/Login';
-import './guest-welcome.css';
+import React, { useEffect, useState } from 'react';
+import { Typography, Grid, Fab, Card, CardHeader, CardContent } from '@material-ui/core';
+import useNetworkStatus from 'hooks/useNetworkStatus';
+import { projectName } from 'project-name';
+import PropTypes from 'prop-types';
+import PWAInstructions from 'components/PWAInstructions/index';
+import LoginDialogue from '../../../../components/LoginDialogue';
```

Comment:
You can use an absolute import here for consistency
```suggestion
import LoginDialogue from 'components/LoginDialogue';
```